README.md

```diff
@@ -219,3 +219,10 @@ if(openPMD_FOUND)
     target_link_libraries(YourTarget PRIVATE openPMD::openPMD)
 endif()
 ```
+
+*Alternatively*, add the openPMD-api repository directly to your project and add it via:
+```cmake
+add_subdirectory("path/to/source/of/openPMD-api")
+
+target_link_libraries(YourTarget PRIVATE openPMD::openPMD)
```

Comment:
Just to see if I understand CMake here:
Assuming ```add_subdirectory``` does only add the path without scanning anything inside, ```target_link_libraries``` will do the scan all subdirectories (if no library ```openPMD::openPMD``` is available yet), find and scan our ```CMakeLists.txt```, find the desired ```openPMD::openPMD``` library, build it and then link it?

Damn, CMake can do _a lot_.

Reply:
`add_subdirectory` will try to access the `CMakeLists.txt` inside its argument.

From there on, a *separate*, *isolated* build for openPMD-api is done (which does not pollute the project's CMake vars that does the `add_subdirectory`). Often such projects then even expose their dependencies as `IMPORTED` targets again.

Reply:
Ah well, [```add_subdirectory```](https://cmake.org/cmake/help/latest/command/add_subdirectory.html) does the heavy lifting.
Still impressive.

Reply:
Yep, modern CMake is awesome :-)